### DISCUSSION TITLE
Ranked Choice Voting Visualiser

### DISCUSSION THREAD

**Discussion by sshmatrix** [snapshot-labs/snapshot#3355] [Show and tell]
Hi Snapshot Labs,

I have created a Ranked Choice Voting Sankey Diagram Visualiser hooked to Snapshot GraphQL API. Currently, users can visualise any Ranked Choice Voting (RCV) proposal on Snapshot with this tool. The app is currently hosted at https://anti-pro.eth.limo. I welcome feedback and happy to incorporate the visualisation into Snapshot UI. 

<img width="728" alt="Screenshot 2022-11-27 at 11 56 00" src="https://user-images.githubusercontent.com/19473027/204131585-ed4e77b5-d4b3-4774-92e9-ee1b3ce5688e.png">

**Comment by samwhosung** [comment]
This is awesome great job! @sshmatrix  Would it be possible to populate the form through the URL? f.e `https://anti-pro.eth.limo/?space=balancer.eth?id=0xfd10343cb4cc23fa126e6c27138d06370638f2223c2a517e078661bf3c2f91d6`

@bonustrack I was thinking we could integrate this a bit like we do with signator.io. For ranked choice proposal we could show a button in the results linking to this more advanced visualization.

**Reply by sshmatrix** [reply]
> This is awesome great job! @sshmatrix Would it be possible to populate the form through the URL? f.e https://anti-pro.eth.limo/?space=balancer.eth?id=0xfd10343cb4cc23fa126e6c27138d06370638f2223c2a517e078661bf3c2f91d6

Absolutely. I will make some code changes to this effect. I'll add some other features to see logs of calculations for more curious users.

> @bonustrack I was thinking we could integrate this a bit like we do with signator.io. For ranked choice proposal we could show a button in the results linking to this more advanced visualization.

That sounds like a good idea and the most straightforward integration at the moment
